source/meta.tex

```diff
@@ -2667,12 +2671,20 @@
 \rSec2[ratio.si]{SI types for \tcode{ratio}}
 
 \pnum
+\indexlibraryglobal{quecto}%
+\indexlibraryglobal{ronto}%
 \indexlibraryglobal{yocto}%
 \indexlibraryglobal{zepto}%
 \indexlibraryglobal{zetta}%
 \indexlibraryglobal{yotta}%
-For each of the \grammarterm{typedef-name}{s} \tcode{yocto}, \tcode{zepto},
-\tcode{zetta}, and \tcode{yotta}, if both of the constants used in its
+\indexlibraryglobal{ronna}%
+\indexlibraryglobal{quetta}%
+For each of the \grammarterm{typedef-name}{s}
+\tcode{quecto}, \tcode{ronto},
+\tcode{yocto}, \tcode{zepto},
+\tcode{zetta}, \tcode{yotta},
+\tcode{ronna}, and \tcode{quetta},
+if both of the constants used in its
 specification are representable by \tcode{intmax_t}, the typedef is
 defined; if either of the constants is not representable by \tcode{intmax_t},
 the typedef is not defined.
```

Comment:
Pre-existing: it seems better to say "declared" instead of "defined" which is more accurate for typedef-names ([[basic.def]/2](https://eel.is/c++draft/basic.def#2)).

Reply:
Oh, I think change should be made in another PR now. There're many occurences of phrases like "certain _typedef-name_ is defined" in the library wording. I'll try to open a PR for this.